src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -212,9 +211,13 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
         Driver driver = new Driver();
```

Comment:
You can use constructor with parameters instead of default constructor and setter.
Please also check the similar cases in other DAO classes.